frontend/src/GameReportForm.tsx

```diff
@@ -435,3 +468,110 @@ function GameReportForm() {
 }
 
 export default GameReportForm;
+
+async function submit(validatedResult: ValidFormData) {
+    return await axios.post(
+        //"http://localhost:8081/submitReport",
+        "https://api.waroftheringcommunity.net:8080/submitReport",
+        toPayload(validatedResult),
+        { headers: { "Content-Type": "application/json" } }
+    );
+}
+
+function toPayload(formData: ValidFormData): GameReportPayload {
+    return {
+        winner: formData.winner.value,
+        loser: formData.loser.value,
+        side: formData.side.value,
+        victory: formData.victory.value,
+        match: formData.match.value,
+        competition: formData.competition.value,
+        league: formData.league.value,
+        expansions: formData.expansions.value,
+        treebeard: formData.treebeard.value,
+        actionTokens: formData.actionTokens.value,
+        dwarvenRings: formData.dwarvenRings.value,
+        turns: formData.turns.value,
+        corruption: formData.corruption.value,
+        mordor: formData.mordor.value,
+        initialEyes: formData.initialEyes.value,
+        aragornTurn: formData.aragornTurn.value,
+        strongholds: formData.strongholds.value,
+        interestRating: formData.interestRating.value,
+        comment: formData.comment.value,
+    };
+}
+
+function toErrorMessage(error: ServerErrorBody): string {
+    if (error.status === 422) {
+        const parsedResult = parseValidationResult(error.response.data);
+        if (parsedResult.length) {
+            const errorMessage = parsedResult
+                .map(validationErrorToMessage)
+                .join(", ");
+            return (
+                errorMessage.slice(0, 1).toUpperCase() + errorMessage.slice(1)
+            );
+        }
+    }
+    return "Something went wrong.";
```

Comment:
Do we want to put contact information in these error messages??

Reply:
Haha totes, let's stick an email in there, what's good?

Reply:
My personal email works, `sirrus233@gmail.com`

Reply:
No, better idea: Let's point them to the #ladder-support channel on the discord!

Reply:
Perfect 💯 grabbed link from you but gonna do this in a follow-up PR so I can rebase the other branch easier